CHANGES

```diff
@@ -22,9 +22,28 @@ Incompatible changes
 Deprecated
 ----------
 
+* #10028: jQuery and underscore.js will no longer be automatically injected into
+  themes from Sphinx 6.0. If your theme's JavaScript uses the ``jQuery``,
```

Comment:
```suggestion
  themes from Sphinx 5.0. If your theme's JavaScript uses the ``jQuery``,
```

Reply:
And... it's not just themes; but extensions as well.

Reply:
From @tk0miya / @TimKam's comment above, I thought a more gradual deprecation was wanted, not actually stopping providing the files until v6. Very happy to bring forwards to v5, but want to make sure there are no surprises!

Reply:
> extensions as well

updated note to include extensions